raiden/tests/unit/storage/test_serialization.py

```diff
@@ -140,7 +140,10 @@ def test_restore_queueids_to_queues(chain_state, netting_channel_state):
     chain_state.queueids_to_queues[queue_identifier] = messages
 
     serialized_chain_state = JSONSerializer.serialize(chain_state)
-
     deserialized_chain_state = JSONSerializer.deserialize(serialized_chain_state)
 
+    for f in dir(chain_state):
```

Comment:
Why is this necessary?

Reply:
oops, that is not necessary, I added it while debugging.